### DISCUSSION TITLE
Definitions or examples of fields/roles

### DISCUSSION THREAD

**Discussion by csidirop** [firefly-iii/firefly-iii#9446] [Firefly III] [answered]
Hi there, I'm using the data importer and created my config by try-and-error matching of those fields. Later I realized that I missed the [doc section about roles](https://docs.firefly-iii.org/references/data-importer/roles/)

But do we have more complete and detailed descriptions of those roles (eg. `Budget name` or `Internal reference`. What is the difference between `Opposing account ID (matching FF3)` and `Opposing account (account number)`). 

Some example data would also be really nice. 

Thanks!

**Comment by JC5** [comment, marked as answer]
Hey, that's a great idea. I'll see if I can't expand that page.

**Reply by JC5** [reply]
Done, let me know if this is what you expected.

**Reply by csidirop** [reply]
Looks great, thank you. Ill try to work that.
